README.md

```diff
@@ -317,6 +317,25 @@ be deleted by the role.
 
 Slaves to bridge/bond/team devices cannot specify `ip` settings.
 
+### `type: ethernet`
+
+Ethernet-specific options can be set using the connection profile variable `ethernet`. This
+variable should be specified as a dictionary with the following items (options): `autoneg`, `speed` and `duplex`,
+which correspond to the settings of the `ethtool` utility with the same name. `speed` is an
+integer giving the speed in Mb/s, the valid values of `duplex` are `half` and `full` and
```

Comment:
As far as I understand, there needs to be a comma before the `and`:
https://english.stackexchange.com/questions/30516/should-i-use-a-comma-before-and-or-or